be/src/common/config.h

```diff
@@ -536,6 +536,15 @@ namespace config {
     // Whether to continue to start be when load tablet from header failed.
     CONF_Bool(ignore_load_tablet_failure, "false");
 
+    // Set max cache's size of query results, the unit is M byte
+    CONF_Int32(cache_max_size, "256"); 
```

Comment:
```suggestion
    CONF_Int32(query_cache_max_size_mb, "256"); 
```

same suggestion for other 2 configs.